site/en/blog/new-in-chrome-101/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: New in Chrome 101
+description: >
+  Chrome 101 is rolling out now with a new method of specifying color with hwb notation, and priority hints giving a way to hint to the browser the ideal order to download resources. And there's plenty more!
+layout: 'layouts/blog-post.njk'
+date: 2022-05-03
+authors:
+  - rachelandrew
+hero: 'image/kheDArv5csY6rvQUJDbWRscckLr1/9pkNxkQXGyaqlEMbLKPO.png'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-101
+---
+
+Here's what you need to know:
+
+- [The `hwb()` color notation](#hwb) gives you a new way to specify color according to hue, whiteness, and blackness.
+- [Priority Hints](#priority-hints) give you a way to hint to the browser in which order resources should be downloaded.
+And there's plenty more.
+
+Let's take a look at what's available in Chrome 101.
+
+
+## `hwb()` color notation {: #hwb }
+
+Described [in an article by Stefan Judis](https://www.stefanjudis.com/blog/hwb-a-color-notation-for-humans/) as a "color notation for humans", [`hwb()`](https://developer.mozilla.org/docs/Web/CSS/color_value/hwb) color specifies color according to hue, whiteness, and blackness. As with other color notations, an optional alpha component specifies opacity.
+
+```css
+h1 {
+  color: hwb(194 0% 0% / .5) /* #00c3ff with 50% opacity */
+}
+```
+
+This method of specifying color is now well-supported, with Firefox supporting it from version 96, and Safari from version 15.
+
+## Priority Hints {: #priority-hints }
+
+Priority Hints give you a way to hint to the browser which order resources should be downloaded in, by using the `fetchpriority` attribute. This accepts values of `"high"`, `"low"`, and `"auto"`. 
+
+`"high"`: You consider the resource a high priority and want the browser to prioritize it as long as the browser's heuristics don't prevent that from happening.
+`"low"`: You consider the resource a low priority and want the browser to deprioritize it if its heuristics permit.
+`"auto"`: This is the default value where you don't have a preference and let the browser decide the appropriate priority.
```

Comment:
```suggestion
*  `"high"`: You consider the resource a high priority and want the browser to prioritize it as long as the browser's heuristics don't prevent that from happening.
*  `"low"`: You consider the resource a low priority and want the browser to deprioritize it if its heuristics permit.
*  `"auto"`: This is the default value that lets the browser decide the appropriate priority.
```

Reply:
This appears as though it should have been a list, but wasn't formatted with bullets.